javascript/javascript3/week1/weatherApp.js

```diff
@@ -0,0 +1,171 @@
+// All Buttons from html
+
+const input = document.querySelector('input')
+const buttonWithCity = document.querySelector('#cityButton')
+const locationButton = document.getElementById('locationButton')
+const saveLocationButton = document.getElementById('saveButton')
+const deleteLocationButton = document.getElementById('deleteLocationButton')
+
+// All class or id's from html
+
+const city = document.querySelector('#cityText')
+const temprature = document.getElementById('tempratureText')
+const feelsLike = document.getElementById('feelsLike')
+const minMaxTemp = document.getElementById('tempMinMax')
+const pressure = document.getElementById('pressure')
+const img = document.querySelector('img')
+const wind = document.getElementById('windSpeed')
+const cloud = document.getElementById('clouds')
+const sunTimes = document.getElementById('sunRiseSet')
+const mapDiv = document.getElementById('mapLocation')
+const mapLocation = document.querySelector('iframe')
+
+// Save Location and Delete Location Events
+
+saveLocationButton.addEventListener('click', () => {
+    localStorage.setItem('city', `${input.value}`)
+    location.reload()
+})
+
+deleteLocationButton.addEventListener('click', () => {
+    localStorage.removeItem('city')
+    location.reload()
+})
+
+// when you save your location, page is automatically opening with your location
+
+if (localStorage.getItem('city') != undefined) {
+    function displayTheDatawithCityWithAutomatic() {
```

Comment:
Be sure to be consistent with the conventions you use - you do a good job of using camelCasing in most places.
```suggestion
    function displayTheDataWithCityWithAutomatic() {
```

Reply:
Wow I missed that letter. I will fix that. Thanks 👍